### ISSUE TITLE
Feature request: allow `deschashonly=True` for CLN invoices

### ISSUE THREAD

**Issue by callebtc** [breez/breez-sdk#404]
Please allow setting the flag `deschashonly=True` via the SDK so that we can create LNURL invoices.

**Comment by JssDWt** [comment]
Related: https://github.com/breez/breez-sdk/issues/367

**Comment by callebtc** [comment]
@JssDWt agreed, fully exposing the flags for the invoice call would be *a lot* more useful then only this flag. It will pave the way for atomic swaps or hodl invoices for example.
